views/partials/prs.hbs

```diff
@@ -46,5 +46,18 @@
             {{/each}}
         </div>
     {{/if}}
+
+    <div class="bg-white br2 center mt5 shadow-1 overflow-hidden w-50-ns w-80-m w-90 mt2">
```

Comment:
Nice one for using the tachyons classes! How about losing the white background here? I think it is secondary to the results and should be a bit more subtle in appearance.

Reply:
Should the color be something different, or just invert the text colors to show up on a dark background?